recipes/mupdf/Makefile-shared-libs-osx.patch

```diff
@@ -0,0 +1,39 @@
+--- Makefile.orig	2020-03-25 11:02:34.874709800 -0500
++++ Makefile	2020-04-07 09:39:20.576540717 -0500
+@@ -20,7 +20,7 @@
+ # Do not specify CFLAGS or LIBS on the make invocation line - specify
+ # XCFLAGS or XLIBS instead. Make ignores any lines in the makefile that
+ # set a variable that was set on the command line.
+-CFLAGS += $(XCFLAGS) -Iinclude
++CFLAGS += $(XCFLAGS) -Iinclude -fPIC
```

Comment:
Add `-fPIC` to `XCFLAGS` ?

Reply:
This is part of an upstream patch, so I don't want to change it.